MSProject/ViewControllers/LibraryTableView/WBLibraryTableViewController.swift

```diff
@@ -130,16 +130,33 @@ class WBLibraryTableViewController: UIViewController {
             switch result {
             case .success(let value):
                 self.viewModel.state.value = value.isEmpty ? ViewState.empty : ViewState.value
+                self.loadRentsAndWishes()
             case .failure(let error):
                 self.showAlert(message: error.localizedDescription)
                 self.viewModel.state.value = ViewState.error
             }
         }
-//        viewModel.bookViewModels.signal.observeValues { (_) in
-//            self._view.bookTable.reloadData()
-//        }
     }
-            
+    
+    private func loadRentsAndWishes() {
+        viewModel.loadRents().startWithResult { [unowned self] result in
+            switch result {
+            case .success:
+                self._view.bookTable.reloadData()
+            case .failure(let error):
+                print(error)
+            }
+        }
+        viewModel.loadWishes().startWithResult { [unowned self] result in
```

Comment:
if you are not going to use the error value, or if you want to subscribe to the error separatedly, you can use .startWithValues